moda-contracts/test/SplitsFactory.t.sol

```diff
@@ -0,0 +1,192 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.21;
+
+import {SplitsFactory} from "../src/SplitsFactory.sol";
+import {ISplitsFactory} from "../src/interfaces/ISplitsFactory.sol";
+import {IManagement} from "../src/interfaces/IManagement.sol";
+import {ISplitMain} from "../src/interfaces/0xSplits/ISplitMain.sol";
+import {IOfficialModaContracts} from "../src/interfaces/IOfficialModaContracts.sol";
+import {SplitMainMock} from "./mocks/SplitMainMock.sol";
+import {ModaTest} from "./utils/ModaTest.sol";
+
+contract ModaRegistry is IOfficialModaContracts {
```

Comment:
Maybe we can set this up as a fork test? 

https://book.getfoundry.sh/forge/fork-testing#forking-cheatcodes